README.md

```diff
@@ -547,10 +547,14 @@ Once configured correctly you should be able to do a multi-hop SSH via a jumphos
   unset JUMPHOST_USER
   ansible-playbook -u "${lor_admin_user}" -l '!jumphost' cluster.yml
   ```
-* (Re-)deploying only a specific role - e.g. *rsyslog_client* - on the previously deployed test cluster *Talos*
+* (Re-)deploying only a specific role - e.g. *rsyslog_client* - on a previously deployed stack
   ```bash
   ansible-playbook -u "${lor_admin_user}" single_role_playbooks/rsyslog_client.yml
   ```
+* (Re-)deploying only specific tasks with a certain tag from multiple roles - e.g. those with the tag *groups* to rerun relevant tasks when a group was added or its config was changed - on a previously deployed stack
+  ```bash
+  ansible-playbook -u "${lor_admin_user}" single_tag_playbooks/groups.yml -t groups
```

Comment:
Why do we need specific playbook directory for using tags? Tags work on existing single_role_playbooks just fine. It is further complicating the LoR and I don't see any added value. What am I missing.

Reply:
Point taken, but you somehow need to have a list to know what tags are available in what roles. This `groups` playbook runs only the relevant roles, which have that tag. You could re-run all roles and specify the same tag, but that would take a lot longer and clutter the output with even more "skipped" tasks making it harder to find relevant output.

Reply:
If there is a better way to handle such cases of rerunning a subset, which is more than a single role (in a specific order) and less than everything, then we can also use that... The groups.yml playbook prevents admins from forgetting the re-run a role required when something changed for a group and it prevents admins from re-running the individual single role playbooks in the wrong order.

Reply:
In fact just had to re-run several roles in order to add a new group to Vaxtron and noticed I forgot to add the `sudoers` role, which is required before re-running the `shared_storage` role -> added in extra commit.

Reply:
I like the idea of a commandline `help` feature/option; that documents available tags for a single role, but does not ensure a subset of tasks from a subset of roles is executed in the correct order.